Sources/StreamChatUI/ChatMessageList/ChatMessageListRouter.swift

```diff
@@ -42,4 +42,13 @@ open class _ChatMessageListRouter<ExtraData: ExtraDataTypes>: ChatRouter<_ChatMe
         let navigation = UINavigationController(rootViewController: preview)
         rootViewController.present(navigation, animated: true)
     }
+    
+    open func showImageGallery(for message: _ChatMessage<ExtraData>) {
```

Comment:
I think we should also pass the index of the image the user tapped on because that should be the one that you see first.